Core2/src/main/res/values-de/strings.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?><!--
+  ~ Infomaniak SwissTransfer - Android
```

Comment:
Should we rename `SwissTransfer` into `Core2` in these headers ?

Reply:
Yeah good question, I wondered about this as well, but I thought that for now as this is still a submodule of SwissTransfer it would be more appropriate and easier to keep `SwissTransfer` and change it all at the same time the day we extract it in its own repository

Reply:
Ok for me 👌